tests/Moq.Tests/Regressions/FluentMockVisitorIssues.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Xunit;
```

Comment:
Right now, we have one single file for all regression tests coming from GitHub issues; `IssueReportsFixture.cs`. Usually, there's a `#region <IssueNumber>" for every issue (they are sorted by source & issue number), and in that region there's a `public class Issue<IssueNumber>` containing all tests related to that issue.

Could you please move these tests there? Thanks!

(Btw., the following hint is not relevant if you move your tests to `IssueReportsFixture.cs`, but if you create a new code file anywhere in the Moq solution, you should also add the copyright notice at the very beginning of the new file. You can usually just copy & paste it from any other code file:)

```diff
+// Copyright (c) 2007, Clarius Consulting, Manas Technology Solutions, InSTEDD.
+// All rights reserved. Licensed under the BSD 3-Clause License; see License.txt.
+
```